Engine/Rumor.cs

```diff
@@ -432,7 +434,8 @@ public void Choose(int index)
 			if (0 <= index && index < State.Consequences.Count) {
 				EnterBlock(State.Consequences[index]);
 				LastChoice = State.Choices[index];
-				State.RemoveChoice(index);
+				LastChoiceIndex = index;
+				State.ClearChoices();
```

Comment:
I don't think the state should be cleared of all choices every time you select a choice. I have several reasons for believing this:

* The `clear choices` command already exists, and it removes all choices
* There is an undocumented feature for the command `choose` that allows you to specify how many choices the user needs to pick before they can continue, and it won't work if you clear all of the choices after choosing just one choice

I think there is a better way to implement this kind of feature if you want it by default, perhaps as a default function bind or as an additional argument to `choose`. What do you think?

Reply:
Ah, I forgot I'd added this. The way I was using Rumor was to jump to an entirely new label with new text and choices, so it made sense to always remove all choices. I can see you might want someone to exhaust the choices one by one though. I could have implemented this change by responding to OnChoice, so I don't think it's something that Rumor definitely needs, but it might be nice to have a mode for this. I'll just remove this change from my PR.